DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoBasicConnectorBinding.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rhino;
+using Rhino.DocObjects;
+using Rhino.Geometry;
+using Sentry.Reflection;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models;
+using Speckle.Connectors.DUI.Models.Card;
+using Speckle.Connectors.Rhino7.Extensions;
+using Speckle.Connectors.Rhino7.HostApp;
+
+namespace Speckle.Connectors.Rhino7.Bindings;
+
+public class RhinoBasicConnectorBinding : IBasicConnectorBinding
+{
+  public string Name { get; set; } = "baseBinding";
+  public IBridge Parent { get; set; }
+  public BasicConnectorBindingCommands Commands { get; }
+
+  private readonly DocumentModelStore _store;
+  private readonly RhinoSettings _settings;
+
+  public RhinoBasicConnectorBinding(DocumentModelStore store, RhinoSettings settings, IBridge parent)
+  {
+    _store = store;
+    _settings = settings;
+    Parent = parent;
+    Commands = new BasicConnectorBindingCommands(parent);
+
+    _store.DocumentChanged += (_, _) =>
+    {
+      Commands.NotifyDocumentChanged();
+    };
+  }
+
+  public string GetConnectorVersion() =>
+    typeof(RhinoBasicConnectorBinding).Assembly.GetNameAndVersion().Version ?? "No version";
+
+  public string GetSourceApplicationName() => _settings.HostAppInfo.Slug;
+
+  public string GetSourceApplicationVersion() => "7";
```

Comment:
We might have some of this in Revit (in which case it should come out) but I'd move as much to the settings as possible as a connection to a settings class is cleaner than a connection to the RhinoBasicConnectorBinding

Reply:
Oh good catch! I didn't realise this was hardcoded here, we should definitely move it to settings.